tests/test_ycsb.sh

```diff
@@ -40,12 +40,15 @@ spec:
    role: mongo
 EOF
 cat << EOF | kubectl apply -f -
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
this is a duplicate of https://github.com/cloud-bulldozer/ripsaw/pull/275.

Reply:
yes, however I was updating them all and it fails if its not updated properly. So if we merge the other one first thats fine. If not then we'll catch them all in one shot.